gp-templates/translations.php

```diff
@@ -75,19 +75,39 @@
 	<?php
 	$filter_links = array();
 
+	$filters_and_sort = array_merge( array_filter( $filters ), array_filter( $sort ) );
+
+	$all_filters = array(
+		'status' => 'current_or_waiting_or_fuzzy_or_untranslated',
+	);
+
+	$bold_on = array(
+		'open' => '<b>',
+		'close' => '</b>',
+	);
+
+	$bold_off = array(
+		'open' => '',
+		'close' => '',
+	);
+
+	array() === array_diff( $all_filters, $filters_and_sort ) || array() === $filters_and_sort ? $add_bold = $bold_on : $add_bold = $bold_off;
+
 	// Translators: %s is the total strings count for the current translation set.
-	$filter_links[] = gp_link_get( $url, sprintf( __( 'All&nbsp;(%s)', 'glotpress' ), number_format_i18n( $translation_set->all_count() ) ) );
+	$filter_links[] = gp_link_get( $url, $add_bold['open'] . sprintf( __( 'All&nbsp;(%s)', 'glotpress' ) . $add_bold['close'], number_format_i18n( $translation_set->all_count() ) ) );
```

Comment:
`gp_link_get()` supports a third argument to pass additional attributes. Could we use that instead of the cryptic _bold_ variables? :) 

Something like this:
```php
$is_current = array() === array_diff( $all_filters, $filters_and_sort ) || array() === $filters_and_sort
$filter_links[] = gp_link_get(
	$url,
	sprintf( … ),
	$is_current ? array( 'class' => 'current' ) : array()
);
```